Binaries for linux-arm

### Expected behavior

Find a `skaffold-linux-arm` binary from [the releases page](https://github.com/GoogleContainerTools/skaffold/releases).

### Actual behavior

There is no `skaffold-linux-arm` binary.

### Information

- Skaffold version: v0.28.0
- Operating system: Arch Linux ARM on Raspberry Pi 3B+ (armhf)

Thanks.

EDIT: cross compiling `skaffold` with `GOOS=linux GOARCH=arm GOARM=5` seems to work very well.

```
root@657e2b8716d2:/go/src/github.com/GoogleContainerTools/skaffold# git apply
diff --git a/Makefile b/Makefile
index 1f6a11ad..d2cdc804 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ GCP_PROJECT ?= k8s-skaffold
 GKE_CLUSTER_NAME ?= integration-tests
 GKE_ZONE ?= us-central1-a

-SUPPORTED_PLATFORMS := linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
+SUPPORTED_PLATFORMS := linux-arm linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
 BUILD_PACKAGE = $(REPOPATH)/cmd/skaffold

 VERSION_PACKAGE = $(REPOPATH)/pkg/skaffold/version
root@657e2b8716d2:/go/src/github.com/GoogleContainerTools/skaffold# make cross
make: *** No rule to make target 'out/skaffold-linux-arm', needed by 'out/skaffold-linux-arm.sha256'.  Stop.
```

So, it does not work. However the following works for `skaffold-linux-arm`, but obviously it's not the correct patch:

```
root@657e2b8716d2:/go/src/github.com/GoogleContainerTools/skaffold# git apply
diff --git a/Makefile b/Makefile
index 1f6a11ad..f93b17a1 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 GOOS ?= $(shell go env GOOS)
-GOARCH = amd64
+GOARCH = arm
 BUILD_DIR ?= ./out
 ORG := github.com/GoogleContainerTools
 PROJECT := skaffold
@@ -28,7 +28,7 @@ GCP_PROJECT ?= k8s-skaffold
 GKE_CLUSTER_NAME ?= integration-tests
 GKE_ZONE ?= us-central1-a

-SUPPORTED_PLATFORMS := linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
+SUPPORTED_PLATFORMS := linux-arm linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
 BUILD_PACKAGE = $(REPOPATH)/cmd/skaffold

 VERSION_PACKAGE = $(REPOPATH)/pkg/skaffold/version
root@657e2b8716d2:/go/src/github.com/GoogleContainerTools/skaffold# make cross
GOOS=linux GOARCH=arm CGO_ENABLED=0 go build -ldflags " -extldflags \"\" -X github.com/GoogleContainerTools/skaffold/pkg/skaffold/version.version=v0.28.0-80-g25f1b664-dirty -X github.com/GoogleContainerTools/skaffold/pkg/skaffold/version.buildDate=2019-05-06T19:20:21Z -X github.com/GoogleContainerTools/skaffold/pkg/skaffold/version.gitCommit=25f1b66402507440a12a0c1173806b190ff76eb8 -X github.com/GoogleContainerTools/skaffold/pkg/skaffold/version.gitTreeState=dirty " -gcflags "all=-trimpath=/go/src/github.com/GoogleContainerTools/skaffold" -asmflags "all=-trimpath=/go/src/github.com/GoogleContainerTools/skaffold" -tags "kqueue" -o out/skaffold-linux-arm github.com/GoogleContainerTools/skaffold/cmd/skaffol
...
(And then darwin-arm fails)
```

Gave it a shot:

```
hack/generate-statik.sh
Collecting licenses
Error: go [list -e -json -compiled=false -test=false -export=false -deps=true -find=false -- github.com/GoogleContainerTools/skaffold/cmd/skaffold]: exit status 1: build github.com/Azure/go-autorest/tracing: cannot find module for path github.com/Azure/go-autorest/tracing

Usage:
  licenses save <package> [flags]

Flags:
      --force              Delete the destination directory if it already exists.
  -h, --help               help for save
      --save_path string   Directory into which files should be saved that are required by license terms

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --confidence_threshold float       Minimum confidence required in order to positively identify a license. (default 0.9)
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

F0217 08:13:49.776454    4073 main.go:43] go [list -e -json -compiled=false -test=false -export=false -deps=true -find=false -- github.com/GoogleContainerTools/skaffold/cmd/skaffold]: exit status 1: build github.com/Azure/go-autorest/tracing: cannot find module for path github.com/Azure/go-autorest/tracing
make: *** [Makefile:228: generate-statik] Error 1```

--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 GOOS ?= $(shell go env GOOS)
-GOARCH = amd64
+GOARCH ?= amd64
 BUILD_DIR ?= ./out
 ORG = github.com/GoogleContainerTools
 PROJECT = skaffold

--- a/Makefile.diag
+++ b/Makefile.diag
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 GOOS ?= $(shell go env GOOS)
-GOARCH = amd64
+GOARCH ?= amd64
 BUILD_DIR ?= ./out
 ORG := github.com/GoogleContainerTools
 PROJECT := diag

